verticaltabs.jsm

```diff
@@ -438,6 +463,7 @@ VerticalTabs.prototype = {
       tabs.removeEventListener('TabPinned', this, false);
       tabs.removeEventListener('TabUnpinned', this, false);
       tabs.removeAttribute('vertical');
+      tabs.removeChild(document.getElementById('filler-tab'));
```

Comment:
Should we be removing the `filler-tab`?  I was hoping XUL would handle that for us…